source/reference/mongosync-states.txt

```diff
@@ -85,3 +85,14 @@ operations in that state.
      - - ``GET`` :ref:`/progress <c2c-api-progress>`
        - ``GET`` :ref:`/reverse <c2c-api-reverse>`
 
+   * - .. _c2c-state-reversing:
+
+       ``REVERSING``
+
+     - The sync process copies metadata from the destination
+       cluster to the source cluster. Then, MongoDB swaps the
+       source and destination clusters and resumes applying
+       change events.
+     - - ``POST`` :ref:`/reverse <c2c-api-reverse>`
+       - ``GET`` :ref:`/progress <c2c-api-progress>`
```

Comment:
```suggestion
     - - ``GET`` :ref:`/progress <c2c-api-progress>`
```

I just noticed `reverse` is not a possible command under this state.

Reply:
Fixed